P/Poppler/build_tarballs.jl

```diff
@@ -77,21 +75,20 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    BuildDependency(PackageSpec("CMake_jll", v"3.22.2")), # we need 3.22.0
```

Comment:
Use `HostBuildDependency`
```suggestion
    HostBuildDependency(PackageSpec("CMake_jll", v"3.22.2")), # we need 3.22.0
```